docs/streams/upgrade-guide.html

```diff
@@ -76,8 +76,13 @@ <h3><a id="streams_api_changes_240" href="#streams_api_changes_240">Streams API
 
     <!-- Placeholder KIP-213 -->
     <!-- Placeholder KIP-307 -->
-    <!-- Placeholder KIP-479 -->
+    <p>
+        Before the 2.4.0 version of Kafa Streams, users of the DSL could not name the state stores involved in a stream-stream join.  If users added DSL operations before of the 
+        join, the internal names of the state stores would shift, requiring an application reset when redeploying.  In the 2.4.0 release, Kafka Streams adds the <code>StreamJoined</code>
+        object, which gives users the ability to name the join processor, repartition topic(s) (if a repartition is required), and the state stores involved in the join.  Also, by naming the state stores, the changelog topics backing the state stores are named as well.
 
```

Comment:
Missing `<br /> ` markup ?

Reply:
Is it `<br>` or `<br />`? I've seen both in the docs and both seem to work, but what's the difference?

Reply:
ack